x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/operator/EvalOperator.java

```diff
@@ -55,17 +53,7 @@ public String toString() {
 
     @Override
     public void close() {
-        Releasables.closeExpectNoException(evaluator);
-    }
-
-    /** Returns a copy of the give block, if the block appears in the page. */
-    // TODO: this is a catch all, can be removed when we validate that evaluators always return copies
-    // for now it just looks like Attributes returns a reference?
-    static Block maybeCopyBlock(Page page, Block block) {
-        if (IntStream.range(0, page.getBlockCount()).mapToObj(page::getBlock).anyMatch(b -> b == block)) {
-            return BlockUtils.deepCopyOf(block);
-        }
-        return block;
```

Comment:
do we not need this anymore, since the evaluators will not emit pages with such issues now?

Reply:
We emit them but we detect by asking the reference is if is "floating" - so we never have to check. Still have to deep copy. For now!